Apps/WebClient/src/Properties/launchSettings.json

```diff
@@ -11,11 +11,11 @@
     "HealthGateway.WebClient": {
       "commandName": "Project",
       "launchBrowser": true,
-      "applicationUrl": "http://localhost:5000;",
       "environmentVariables": {
         "DOTNET_USE_POLLING_FILE_WATCHER": "1",
         "ASPNETCORE_ENVIRONMENT": "Development"
-      }
+      },
+      "applicationUrl": "http://localhost:5000;"
```

Comment:
this looks odd. Was it not used correctly before?

Reply:
it was changed by vs/visual studio.